### PR TITLE
Use Tcl/Tk packages on FreeBSD.

### PR DESCRIPTION
**Pull request by yurivict**

### CODE REVIEW

File: deps/build.jl [hunk 1]
```diff
@@ -17,6 +17,8 @@ end
 
 provides(AptGet,"tcl8.5",tcl)
 provides(AptGet,"tk8.5",tk)
+provides(BSDPkg,"tcl85",tcl,os=:FreeBSD)
+provides(BSDPkg,"tk85",tk,os=:FreeBSD)
```
Comment (tkelman) [inline deps/build.jl]: this requires a minimum bindeps version of 0.6.0 https://github.com/JuliaLang/BinDeps.jl/commit/264e086062f3c39adba7997f187bd54bc0e6b218

you'll also need to drop julia 0.5 support as it's not longer supported

**Comment by yurivict** [comment]
I changed to BinDeps 0.6.0.
I'm not sure how to drop julia 0.5 support.

**Comment by iblis17** [comment]
```diff
diff --git a/REQUIRE b/REQUIRE
index 13eee6a..02a62e2 100644
--- a/REQUIRE
+++ b/REQUIRE
@@ -1,6 +1,6 @@
-julia 0.5
+julia 0.6
 Compat 0.17.0
 Cairo
 Graphics 0.1
-BinDeps 0.2.2-
+BinDeps 0.6.0
 @windows WinRPM
```

**Comment by aviks** [comment]
Closed by #148

**Pull request closed without merging**